cEP-0031.md

```diff
@@ -0,0 +1,254 @@
+# Improve Generic Bear Quality
+
+| Metadata |                                             |
+| -------- | ------------------------------------------- |
+| cEP      | 31                                          |
+| Version  | 0.1                                         |
+| Title    | Improve Generic Bear Quality                |
+| Authors  | Bhushan Khanale <mailto:bkhanale@gmail.com> |
+| Status   | Proposed                                    |
+| Type     | Feature                                     |
+
+## Abstract
+
+This cEP describes the improvement in the generic bears currently available
+along with the implementation of the new bears as a part of the
+[GSoC 2019 Project][project].
+
+## Introduction
+
+coala has few generic bears which have the potential to perform better
+fixing some of the issues in them. Some of these issues are as follows:
+
+-   [coala/coala-bears#2092][2092]: Pygmented AnnotationBear
```

Comment:
Where did this one go?  I dont see it being explained below.

This is actually a mini-project in itself.  There are at least two WIP's in coala repo, and maybe on in coala-bears, reagarding Pygments.  It would be wonderful to have someone pick them up and create a new Pygment meta bear like the antlr bear framework at https://gitlab.com/coala/bears

Reply:
Will update this.